server/utils/tasks/listTable.ts

```diff
@@ -93,13 +147,44 @@ const unallocatedTableRows = (tasks: Array<Task>): Array<TableRow> => {
   return rows
 }
 
-const tasksTableRows = (tasks: Array<Task>, allocatedFilter: string): Array<TableRow> => {
-  const rows: Array<TableRow> =
-    allocatedFilter === 'allocated' ? allocatedTableRows(tasks) : unallocatedTableRows(tasks)
+export const completedAtDateCell = (item: Task): TableCell => ({
+  text: item.outcomeRecordedAt ? DateFormats.isoDateTimeToUIDateTime(item.outcomeRecordedAt) : '',
+})
+
+export const completedByCell = (task: Task): TableCell => ({
+  text: task.allocatedToStaffMember?.name,
+})
+
+export const decisionCell = (task: Task): TableCell => ({
+  text: getDecisionOutcome(task),
+})
+
+const completedTableRows = (tasks: Array<Task>): Array<TableRow> => {
+  const rows = [] as Array<TableRow>
+
+  tasks.forEach(task => {
+    rows.push([
+      nameAnchorCell(task),
+      completedAtDateCell(task),
+      completedByCell(task),
+      taskTypeCell(task),
```

Comment:
From the 'After' screenshot it looks like the `Task type` column is empty, it would be good to understand why